packages/x-adapter-platform/src/__tests__/__fixtures__/search.response.ts

```diff
@@ -295,8 +295,6 @@ export const platformSearchResponse = {
         title: 'Banner title',
         url: 'https://assets.empathy.co/',
         tagging: {
```

Comment:
Shouldn't you add the same tagging object to the promoted response?

Reply:
Promoted didn't have any tagging object before (with the tagging&query object), so I understood that maybe it must be like this for some reason. But yes, I think that promoted response should contain the same tagging as banner response.